docs/modules/ROOT/pages/advanced-how-tos/releasing/tenant-release-pipelines.adoc

```diff
@@ -0,0 +1,79 @@
+
+The gist here is that you are going to follow the regular instructions to xref:/advanced-how-tos/releasing/create-release-plan.adoc[create a ReleasePlan], but you will omit the `target` and specify the `pipelineRef` directly on your `ReleasePlan`.
+
+
+.*Prerequisites*
+
+* You have an existing Development workspace.
+* You have completed the steps listed in the xref:/getting-started/cli.adoc[Getting started in the CLI] page.
+* Either choose one of the link:https://github.com/konflux-ci/release-service-catalog[konflux release pipelines], or write your own.
+
+NOTE: If you write a good reusable release pipeline, please submit it to our link:https://github.com/konflux-ci/release-service-catalog[catalog] so others can use it.
+
+.Procedure
+
+. Create a `ReleasePlan.yaml` object locally.
+
++
+*Example `ReleasePlan.yaml` object*
+
++
+[source,yaml]
+----
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleasePlan
+metadata:
+ labels:
+   release.appstudio.openshift.io/auto-release: 'true' <.>
+   release.appstudio.openshift.io/standing-attribution: 'true'
+ name: publish <.>
+ namespace: dev-workspace <.>
+spec:
+ application: <application-name> <.>
+ data:
+   mapping:
+     components: <.>
+       - name: demo-component-1
+         repository: registry/destination-image-repository-1
+         tags: [latest]
+       - name: demo-component-2
+         repository: registry/destination-image-repository-2
+         tags: [latest]
+ pipelineRef: <.>
+   resolver: git
+   params:
+     - name: url
+       value: "https://github.com/konflux-ci/release-service-catalog.git"
+     - name: revision
+       value: production
+     - name: pathInRepo
+       value: "pipelines/push-to-external-registry/push-to-external-registry.yaml"
+ serviceAccont: appstudio-pipeline <.>
+----
+
++
+<.> Optional: Control if Releases should be created automatically for this ReleasePlan when tests pass. Defaults to true.
+<.> The name of the release plan.
+<.> The development team's workspace.
+<.> The name of the application that you want to release via a pipeline in the development workspace.
+<.> A list containing the destination repository for each component
+<.> Reference to the Pipeline to be executed in the development workspace.
+<.> The name of the service account used to execute the tenant pipeline.
+
+. In the Development workspace, apply the `ReleasePlan.yaml` file and add the resource to your cluster by running the following command:
+
++
+[source,shell]
+----
+kubectl apply -f ReleasePlan.yaml -n dev
+----
+
+. Provision any secrets needed by the particular release pipeline you chose and xref:/how-tos/configuring/creating-secrets.adoc[upload them to the Development workspace].
+
+.Next steps
+
+* *Create a `release` object:* The development team creates a Release object to references a specific Snapshot and ReleasePlan. It indicates the users' intent to release that Snapshot via the tenant release pipeline defined in the ReleasePlan.
```

Comment:
Is the UI currently limited in that it doesn't show these pipelines?

Reply:
Yes, that's correct. @davidmogar was asking the UI team on how to proceed, but I'm not sure what the outcome of the discussion was (hopefully a jira).

Reply:
I'm in talks with the UX team and we'll schedule a meeting to address the lack of support for showing our pipelines. There was an initial design to show release pipelines in general and not just the managed one, but no progress was done. Now that we are adding the final pipeline next week, we really need to show this info.